src/app/campaign/store/models/flight-days.models.ts

```diff
@@ -17,6 +17,7 @@ export interface FlightDays {
   flightDoc?: HalDoc;
   flightId: number;
   days: FlightDay[];
+  preview?: boolean;
```

Comment:
Ah!  I was wondering about this - it was at the wrong level of the state before.

Reply:
fixed. forever. was maybe not my best work, ok